scripts/lsp_types.lua

```diff
@@ -18,7 +19,90 @@ local function tofile(fname, text)
   end
 end
 
+local function gen_methods(opt)
+  local uri = 'https://raw.githubusercontent.com/microsoft/language-server-protocol/gh-pages/_specifications/lsp/'
+    .. opt.version
+    .. '/metaModel/metaModel.json'
```

Comment:
Why does this use metaModel.json, but the rest of the script uses lsp.json? Stick with one unless there is an important reason to use both.

Reply:
`lsp.json` is in a lsp python implementation repo. this `metaModel.json` in lsp official repo .

Reply:
This URL does not mention python: https://github.com/neovim/neovim/blob/861ecc0b33a7b90b2187821654cdac44c4b8bcea/scripts/lsp_types.lua#L111

If lsp.json isn't needed, then that part of the script should be updated to use metaModel.json

Reply:
The official metaModel.json should be used uniformly. sorry that repo is `Language Server Protocol types code generator`